pkg/minikube/driver/driver.go

```diff
@@ -144,6 +145,24 @@ func IsDockerDesktop(name string) bool {
 	return false
 }
 
+// IsDockerBTRFS checks if Docker is using btrfs storage driver
+func IsDockerBTRFS(name string) bool {
+	if !IsDocker(name) {
+		return false
+	}
+	cmd := exec.Command("docker", "info", "-f", "{{.Driver}}")
```

Comment:
docker info is an expensive call, for that reason we have a cached version of it.


lets change this to oci.CachedDaemonInfo("docker")
"oci.CachedDaemonInfo"


I suspect this is same for podman, since it uses same base image?

Reply:
Done!